### PR TITLE
Docs: Add `Using bundlers to bundle CKEditor 5` document.

### PR DESCRIPTION
**Pull request by filipsobol**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Docs: Add `Using bundlers to bundle CKEditor 5` document.

---

### Additional information

_For example – encountered issues, assumptions you had to make, other affected tickets, etc._

### CODE REVIEW

**Review by godai78 — DISMISSED** [review]

File: docs/getting-started/setup/using-bundlers.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+---
+category: setup
+meta-title: Using bundlers | CKEditor 5 documentation
+order: 130
+modified_at: 2024-08-30
+---
+
+# Using bundlers to bundle CKEditor&nbsp;5
```
Comment (gorzelinski) [inline docs/getting-started/setup/using-bundlers.md:8]: The heading is shortened in the table of contents. Maybe we could add a menu title or make the heading shorter.
```suggestion
# Using bundlers to bundle the editor.
```

File: docs/getting-started/setup/using-bundlers.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
+---
+category: setup
+meta-title: Using bundlers | CKEditor 5 documentation
+order: 130
```
Comment (gorzelinski) [inline docs/getting-started/setup/using-bundlers.md:4]:
```suggestion
meta-title: Using bundlers | CKEditor 5 documentation
menu-title: Using bundlers
order: 130
```

File: docs/getting-started/setup/using-bundlers.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+## Bundling CKEditor&nbsp;5 with Rollup
+
+To bundle CKEditor&nbsp;5 with Rollup, you need to install the following plugins:
+
+1. `@rollup/plugin-node-resolve` &ndash; to allow Rollup to resolve dependencies.
+2. `@rollup/plugin-commonjs` &ndash; to convert CommonJS modules to ES6.
+3. `rollup-plugin-import-css` or `rollup-plugin-styles` &ndash; to allow Rollup to handle CSS files.
+4. `@rollup/plugin-json` &ndash; to allow Rollup to handle JSON files.
+5. `@rollup/plugin-terser` &ndash; to minify the output.
```
Comment (gorzelinski) [inline docs/getting-started/setup/using-bundlers.md:30]: It's a nitpick, but maybe we could order those the same way they are ordered in the snippet below.

File: docs/getting-started/setup/using-bundlers.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+					MiniCssExtractPlugin.loader,
+					'css-loader'
+				]
+			}
+		]
+	}
+};
+```
+
+Some setups and meta-frameworks based on webpack (such as Next.js), may use `babel-loader` to transpile JavaScript to a lower version. If the version of `babel-loader`, `@babel/core`, and `@babel/preset-env` are recent enough, you should be able to use CKEditor&nbsp;5 without any additional configuration. However, projects using older versions of these packages may encounter the `Module parse failed: Unexpected token` error. It is caused by some ES2022 features that CKEditor&nbsp;5 uses, such as native class properties. To address this issue, you need to add the [`@babel/plugin-proposal-class-properties`](https://babeljs.io/docs/babel-plugin-transform-class-properties) plugin to your Babel configuration.
```
Comment (gorzelinski) [inline docs/getting-started/setup/using-bundlers.md:114]:
```suggestion
Some setups and meta-frameworks based on webpack (such as Next.js), may use a `babel-loader` to transpile JavaScript to a lower version. If the version of `babel-loader`, `@babel/core`, and `@babel/preset-env` are recent enough, you should be able to use CKEditor&nbsp;5 without any additional configuration. However, projects using older versions of these packages may encounter the `Module parse failed: Unexpected token` error. It is caused by some ES2022 features that CKEditor&nbsp;5 uses, such as native class properties. You need to add the [`@babel/plugin-proposal-class-properties`](https://babeljs.io/docs/babel-plugin-transform-class-properties) plugin to your Babel configuration to address this issue.
```

**Review by gorzelinski — COMMENTED** [review]
I added some nitpicks.

File: docs/getting-started/setup/using-bundlers.md (hunk 3 above)
Comment (filipsobol) [reply docs/getting-started/setup/using-bundlers.md]: Done :heavy_check_mark:

**Pull request closed without merging**